### PR TITLE
2020-04-08-Hummingboard-Buildroot-Yocto.md: add post to beta

### PR DESCRIPTION
**Pull request by cezarsob**
Signed-off-by: Cezary Sobczak <cezary.sobczak@3mdeb.com>

### CODE REVIEW

File: blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+  - u-boot
+categories:
+  - Miscellaneous
+  - OS Dev
+
+---
+
+## Introduction
+In this article, I will describe differences between two types of system
+images, i.e. SolidRun and Yocto. I was using a HummingBoard Pulse as a platform
```
Comment (macpijan) [inline blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md:27]: SolidRun is a company (vendor) of the HB boards. It sounds like it is an image type?

File: blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+* Linux at your PC (because this way is easier)
+* 16GB Micro SD card
+* 12V Power adapter (the board has wide-range input of 7V-36V but 12V is
+recommended)
+* MicroUSB to USB for the console because the HummingBoard Pulse has an onboard
+FTDI chip, which means that there is no need to use external UART/USB converter
+* HummingBoard with SOM of course
+
+## Build and run a Buildroot image
+Buildroot was made by a SolidRun company as a target platform for their devices.
```
Comment (macpijan) [inline blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md:47]: It sounds like the Buildroot itself was made by the SolidRun. SolidRun have just added a target for their board to the Buildroot build system.

File: blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md [hunk 3]
```diff
@@ -0,0 +1,772 @@
...
+For iMX8M includes a custom config pre-configured to pull in the latest
+**u-boot** and **Linux kernel** from the SolidRun BSP.
```
Comment (macpijan) [inline blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md:49]: U-Boot shall always be used in a written texts

File: blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md [hunk 4]
```diff
@@ -0,0 +1,772 @@
...
+    sudo minicom -b 115200 -D /dev/ttyUSBx
+    ```
+
+    > Note: you need to connect a MicroUSB port with PC
+
+## Build and run a Yocto image
+SolidRun does not provide Yocto support for **imx8**.
+For this purpose repository `3mdeb/meta-imx8` was created. It contains
+BSP layers for **HummingBoard Pulse** now only, but support for
+other **imx8** platforms will be available in the future.
```
Comment (macpijan) [inline blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md:130]:
```suggestion
other **imx8** could be available in the future.
```

File: blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md [hunk 5]
```diff
@@ -0,0 +1,772 @@
...
+
```
Comment (macpijan) [inline blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md:131]: We might also push support for this board to one of the already existing Yocto meta-layers.

File: blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md [hunk 6]
```diff
@@ -0,0 +1,772 @@
...
+* meta-freescale-distro
+* meta-imx8
+* meta-openembedded
+
+`meta-imx8` is our custom layer, where we made modifications. You can find
+here machines, layers and kernel configurations, BSP and kernel recipes.
+We create or modify a BSP recipes for few elements:
+
+* **ARM Trusted Firmware** for secure boot
+* **u-boot** as a bootloader
```
Comment (macpijan) [inline blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md:149]: U-Boot

File: blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md [hunk 7]
```diff
@@ -0,0 +1,772 @@
...
+For this purpose repository `3mdeb/meta-imx8` was created. It contains
+BSP layers for **HummingBoard Pulse** now only, but support for
+other **imx8** platforms will be available in the future.
+
+As you may know, Yocto Project is using a `bitbake`, so for easier
+management and update components, we are using a tool named `kas`. In more
+details, it is described in earlier [blog](https://blog.3mdeb.com/2019/2019-02-07-kas/)
+and [documentation](https://kas.readthedocs.io/en/1.0/).
+
+For HummingBoard Pulse this layers was used:
```
Comment (macpijan) [inline blog/content/post/2020-04-08-Hummingboard-Buildroot-Yocto.md:137]: Use grammarly.com

File: blog/content/post/2020-04-14-Hummingboard-Buildroot-Yocto.md [hunk 8]
```diff
@@ -0,0 +1,778 @@
...
+  - Yocto
+  - u-boot
+categories:
+  - Miscellaneous
+  - OS Dev
+
+---
+
+## Introduction
+In this article, I will describe differences between two types of system
```
Comment (artur-rs) [inline blog/content/post/2020-04-14-Hummingboard-Buildroot-Yocto.md:26]: @cezarsob please add empty line after each header

**Comment by artur-rs** [comment]
@cezarsob according to this guide: https://github.com/3mdeb/news-and-ideas#add-new-post you should create branch from `develop`, not `master` and set the proper target branch in the PR. I have changed the target branch and right now there are some redundant commits from `master`, you can easily fix this using `git rebase --onto`.

File: blog/content/authors/cezary-sobczak/_index.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Cezary Sobczak"
+identifier: cezary.sobczak
+photo: ""
```
Comment (artur-rs) [inline blog/content/authors/cezary-sobczak/_index.md:4]: @cezarsob please set the path to the photo you have added

**Comment by cezarsob** [comment]
@artur-rs Ok everything is done.

**Comment by artur-rs** [comment]
@macpijan is the post ready for publishing in this week?

**Pull request merged**